eland/plotting/_matplotlib/hist.py

```diff
@@ -16,7 +16,7 @@
 #  under the License.
 
 import numpy as np
-from pandas.core.dtypes.generic import ABCIndexClass
```

Comment:
Pandas renamed `ABCIndexClass` to `ABCIndex`
https://github.com/pandas-dev/pandas/pull/38329